### PR TITLE
Feature/datagrid

### PR DESCRIPTION
**Pull request by rcgillet**

### CODE REVIEW

File: packages/react-magma-dom/src/components/Datagrid/.prettierrc [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+  "tabWidth": 2,
```
Comment (aaronmaturen) [inline packages/react-magma-dom/src/components/Datagrid/.prettierrc:2]: Do you need this file?

File: packages/react-magma-dom/src/components/Datagrid/Datagrid.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,392 @@
+import React from "react";
+import { Datagrid } from ".";
+import { Story, Meta } from "@storybook/react/types-6-0";
+import { DatagridProps } from "./Datagrid";
+import { TableRowColor } from "../Table";
+import {
+  usePagination,
+  UsePaginationControlled,
+  UsePaginationProps,
+} from "../Pagination/usePagination";
```
Comment (aaronmaturen) [inline packages/react-magma-dom/src/components/Datagrid/Datagrid.stories.tsx:10]: This should probably just be exported from index.ts in Pagination?

File: packages/react-magma-dom/src/components/Datagrid/useDatagrid.ts [hunk 3]
```diff
@@ -0,0 +1,20 @@
+/**
```
Comment (aaronmaturen) [inline packages/react-magma-dom/src/components/Datagrid/useDatagrid.ts:1]: If there's no immediate plan for a hook here we can probably delete this

File: packages/react-magma-dom/src/hooks/.prettierrc [hunk 4]
```diff
@@ -0,0 +1,4 @@
+{
```
Comment (aaronmaturen) [inline packages/react-magma-dom/src/hooks/.prettierrc:1]: Is this needed?

File: packages/react-magma-dom/src/index.ts [hunk 5]
```diff
@@ -214,6 +214,9 @@ export { I18nContext } from './i18n';
 export { defaultI18n } from './i18n/default';
 export { I18nInterface } from './i18n/interface';
 export { useMediaQuery } from './hooks/useMediaQuery';
+export { usePagination } from './components/Pagination/usePagination';
```
Comment (aaronmaturen) [inline packages/react-magma-dom/src/index.ts:217]: We should just do `export * from ./components/pagination`

**Review by aaronmaturen — CHANGES_REQUESTED** [review]

File: packages/react-magma-dom/src/components/Datagrid/index.ts [hunk 6]
```diff
@@ -0,0 +1,2 @@
+// export * from "./useDatagrid"
```
Comment (joasmi723) [inline packages/react-magma-dom/src/components/Datagrid/index.ts:1]: Can get rid of commented line, if we're not using this file.

File: website/react-magma-docs/src/pages/api/datagrid.mdx [hunk 7]
```diff
@@ -0,0 +1,985 @@
...
+    <Card isInverse>
+      <Datagrid columns={columns} isInverse rows={rows} />
+    </Card>
+  );
+}
+```
+
+# Datagrid Props
+
+**Any other props supplied will be provided to the wrapping `div` element.**
```
Comment (joasmi723) [inline website/react-magma-docs/src/pages/api/datagrid.mdx:959]:
```suggestion
**Any other props supplied will be provided to the wrapping `table` element.**
```

**Review by joasmi723 — APPROVED** [review]

File: packages/react-magma-dom/src/components/Datagrid/index.ts (hunk 6 above)
Comment (rcgillet) [inline packages/react-magma-dom/src/components/Datagrid/index.ts:1]:
```suggestion
```

**Review by aaronmaturen — APPROVED** [review]

**Pull request merged**